.changelog/755.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+resource/deployment: Add instance_configuration_version field to all resources and allow to update the instance_configuration_id to a value not defined in the template.
```

Comment:
Does it make sense to mention other changes?

Reply:
Yes, it does 🙂 updated in https://github.com/elastic/terraform-provider-ec/pull/755/commits/989b44e01f8ec11092f2d992e533002d8fb1ae3c.